tests/e2e/features/artifacts.feature

```diff
@@ -205,3 +205,68 @@ Feature: Artifacts - UI
     Then Element with "text" "zkSync Era∎ Goerli" should be "visible"
     Then Element with "text" " to successfully receive them " should be "visible"
 
+  @id1389 @contacts
+  Scenario: Check artifacts on the Contacts page
+    Given I am on the Main page
+    When I go to page "/contacts"
+    Then Element with "xpath" "//h1[text()='Contacts']" should be "visible"
+    Then Element with "text" "Add contact" should be "visible"
+    Then Element with "text" "Add contact" should be "clickable"
+    Then Element with "partial class" "small-input-container" should be "visible"
+    Then Element with "placeholder" "Address or ENS or contact name" should be "visible"
+    #add new contact
+    Given I click by "text" with "Add contact" value
+    When I fill the "Name of the contact" input field on the Contacts page with "My second contact" text
+    When I fill the "Ethereum address" input field on the Contacts page with "0x26A4c5Dfe2cA3c9E7E8C417B689F41b6b5745C37" text
```

Comment:
I would suggest to use our actual wallet instead of this old one 0x26A4...C37 --> 0x5aA876bC32BC76EFf5124b19744B5B3C38b35537